lighthouse-core/gather/gatherers/link-elements.js

```diff
@@ -120,11 +127,7 @@ class LinkElements extends Gatherer {
           as: link.as || '',
           crossOrigin: getCrossoriginFromHeader(link.crossorigin),
           source: 'headers',
-          devtoolsNodePath: '',
-          selector: '',
-          nodeLabel: '',
-          boundingRect: null,
-          snippet: '',
+          node,
```

Comment:
We're synthesizing a fake link "element" from response headers. There's no DOM node, naturally.
Feels like we should set this as `null` instead of stubby out a truthy, but empty node.

Will need to tweak the artifact shape, but it seems more descriptive anyhow.